frontend/init/index.css

```diff
@@ -0,0 +1,67 @@
+/**
+ * Display font definitions
+ */
+
+@font-face {
+  font-family: Nova;
+  font-style: normal;
+  font-weight: 300;
+  src: url(./nova-light.woff);
+}
+
+@font-face {
+  font-family: Nova;
+  font-style: normal;
+  font-weight: 600;
+  src: url(./nova-semibold.woff);
+}
+
+/**
+ * Minimal reset
+ */
+
+/* stylelint-disable selector-max-universal */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+/* stylelint-enable */
+
+/**
+ * Foundation proportional font-sizing system
```

Comment:
I found this very hard to read, I wonder if this could be explained more simply.

Reply:
This was copied and pasted straight from the [design-system](https://github.com/barnardos/design-system/blob/master/src/index.css#L32-L41).

Reply:
Cool, I've opened [an issue](https://github.com/barnardos/design-system/issues/124).